ClickToCall.Web/Controllers/CallCenterController.cs

```diff
@@ -29,7 +31,7 @@ public ActionResult Index()
         /// Handle a POST from our web form and connect a call via REST API
         /// </summary>
         [HttpPost]
-        public ActionResult Call(CallViewModel callViewModel)
+        public async Task<ActionResult> Call(CallViewModel callViewModel)
```

Comment:
Perhaps this could be renamed to `CallAsync` ? Someone who really knows their stuff mentioned this to me 😉